XpringKit/XRP/Model/XRPPaymentChannelFund+Org_Xrpl_Rpc_V1_PaymentChannelFund.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+
+/// Conforms to XRPPaymentChannelFund struct while providing an initializer that can construct
+/// an XRPPaymentChannelFund from an Org_Xrpl_Rpc_V1_PaymentChannelFund
+internal extension XRPPaymentChannelFund {
+
+  /// Constructs an XRPPaymentChannelFund from an Org_Xrpl_Rpc_V1_PaymentChannelFund
+  /// - SeeAlso: [PaymentChannelFund Protocol Buffer]
+  /// (https://github.com/ripple/rippled/blob/3d86b49dae8173344b39deb75e53170a9b6c5284/
+  /// src/ripple/proto/org/xrpl/rpc/v1/transaction.proto#L288)
+  ///
+  /// - Parameters:
+  ///     - paymentChannelFund: an Org_Xrpl_Rpc_V1_PaymentChannelFund (protobuf object) whose
+  ///                            field values will be used to construct an XRPPaymentChannelFund
+  /// - Returns: an XRPPaymentChannelFund with its fields set via the analogous protobuf fields.
+  init?(paymentChannelFund: Org_Xrpl_Rpc_V1_PaymentChannelFund) {
+    if paymentChannelFund.hasChannel {
+      self.channel = String(decoding: paymentChannelFund.channel.value, as: UTF8.self)
+    } else {
+      return nil
+    }
+
+    if paymentChannelFund.hasAmount {
+      if let amount = XRPCurrencyAmount(currencyAmount: paymentChannelFund.amount.value) {
+        self.amount = amount
+      } else {
+        return nil
+      }
+    } else {
+      return nil
+    }
```

Comment:
not sure what preferred style is in `XpringKit` but if you invert the top two logical statements and return early, you'll lose a level of indentation ie

```
if !paymentChannelFund.hasChannel {
  return nil
}
self.channel = ....
```

i have strong pref for returning early but up to ya'll

Reply:
Same. 

You can even us a `guard` here which will programmatically enforce you to return early. 

```
guard paymentChannelFund.hasChannel else {
}
```

Reply:
great - i'm going to update this logic throughout the entire set of classes in this PR: https://github.com/xpring-eng/XpringKit/pull/288